app/invites/client/admin/adminInvites.js

```diff
@@ -0,0 +1,59 @@
+import { Template } from 'meteor/templating';
+import { ReactiveVar } from 'meteor/reactive-var';
+import moment from 'moment';
+
+import { t, APIClient } from '../../../utils';
+
+import './adminInvites.html';
+
+Template.adminInvites.helpers({
+	invites() {
+		return Template.instance().invites.get();
+	},
+	daysToExpire() {
+		const { expires, days, createdAt } = this;
+
+		if (days > 0) {
+			if (expires < Date.now()) {
+				return t('Expired');
+			}
+
+			return moment(expires).diff(moment(createdAt), 'days');
```

Comment:
Could use the relative methods like `fromNow` like:
```
moment().endOf('day').fromNow();
```
that would result in `in 13 hours`

That way users would understand when left only some hours to expire.

Reply:
https://momentjs.com/docs/#/displaying/fromnow/